quartz/components/ExplorerNode.tsx

```diff
@@ -16,6 +16,7 @@ export interface Options {
   folderDefaultState: "collapsed" | "open"
   folderClickBehavior: "collapse" | "link"
   useSavedState: boolean
+  usePagePath: boolean
```

Comment:
can you explain why we need this?

Reply:
`usePagePath`, when enabled, will automatically open the explorer folders to the current page. Similar to how breadcrumbs work. I found this to be nice for navigation between nested pages, especially on mobile. Give it a try!

```ts
Component.Explorer({ usePagePath: true }),
```

Reply:
seems like a separate feature rather than this one.

Reply:
Want me to make a separate PR for that?